client/components/ScenariosList/index.jsx

```diff
@@ -157,7 +141,8 @@ ScenariosList.propTypes = {
 
 function mapStateToProps(state) {
     const { isLoggedIn, username } = state.login;
-    return { isLoggedIn, username };
+    const { scenarios } = state.scenario;
+    return { isLoggedIn, username, scenarios };
```

Comment:
I don't quite understand this change

Reply:
I'm getting another piece of state called `scenarios` from `state.scenarios`, returned by the Redux reducer